src/io/fastq.rs

```diff
@@ -3,21 +3,26 @@
 // This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! FastQ reading and writing.
+//! FastQ reading.
 //!
 //! # Example
 //!
 //! ```
 //! use std::io;
 //! use bio::io::fastq;
-//! let reader = fastq::Reader::new(io::stdin());
-//! let records = reader.records().map(|r| r.unwrap());
-//! for record in records {
-//!     let check = record.check();
-//!     if check.is_err() {
-//!         panic!("I got a rubbish record!")
-//!     }
-//!     // your record is ok - do something with it...
+//! let mut reader = fastq::Reader::new(io::stdin());
+//! let mut record = fastq::Record::new();
+//! reader.read(&mut record).expect("Failed to parse record");
+//! while !record.is_empty() {
+//!   if record.is_empty() {
+//!     break;
+//!   }
```

Comment:
This `if` statement is not needed now as it is the condition that the `while` loop operates on

Reply:
Good catch, simply forgot to remove that one...